### PR TITLE
[15.0][MIG] base_kanban_stage_state

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Depends on:
- [x] https://github.com/OCA/server-tools/pull/2488 

No changes needed.

cc @ForgeFlow

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
@AaronHForgeFlow you can rebase this PR

File: base_kanban_stage_state/models/base_kanban_stage.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        return [
+            ("draft", "New"),
+            ("open", "In Progress"),
+            ("pending", "Pending"),
+            ("done", "Done"),
+            ("cancelled", "Cancelled"),
+            ("exception", "Exception"),
+        ]
+
+    state = fields.Selection(_get_states, string="State")
```
Comment (MiquelRForgeFlow) [inline base_kanban_stage_state/models/base_kanban_stage.py]:
```suggestion
    state = fields.Selection(_get_states)
```

File: base_kanban_stage_state/tests/test_base_kanban_stage.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Specialty Medical Drugstore
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestBaseKanbanStage(TransactionCase):
+    def test_get_states(self):
+        """It should return a list of stages"""
+        test_stage = self.env["base.kanban.stage"].with_context({})
```
Comment (MiquelRForgeFlow) [inline base_kanban_stage_state/tests/test_base_kanban_stage.py]:
```suggestion
        test_stage = self.env["base.kanban.stage"]
```
I think this is not needed.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**